src/components/RequestGroupComposition/Request.vue

```diff
@@ -0,0 +1,287 @@
+<template>
+  <form-panel
+    v-if="!getFromObject(formConfig, ['request', 'panel', 'hide'], false)"
+    :id="'request' + index"
+    :title="getFromObject(formConfig, ['request', 'panel', 'title'], 'Request')"
+    :icon="getFromObject(formConfig, ['request', 'panel', 'icon'], 'fab fa-wpexplorer')"
+    :cancopy="getFromObject(formConfig, ['request', 'panel', 'canCopy'], true)"
```

Comment:
Super duper nitpick - you probably know the standards better than me.

Should these props (`cancopy`, `canremove`) be kebab-case so that in the child components they are addressed as their camelCase equivalent? Or at least be more consistent with the other props like `tooltip-config`?

Reply:
You're right, kebab case is preferred. I updated most places I think but didn't get everywhere. Will update!